README.md

```diff
@@ -136,6 +136,23 @@ spec:
   startup:
     # Startup has the same options as liveness
     path: ..
+  # Cascading Labels can be used if you want every resource created by your application to
+  # have the same labels, including your application. This could for example be useful for
+  # metrics, where a certain label and the corresponding resources liveliness can be combined.
+  # Any amount of labels can be added as wanted.
+  cascadingsLabels:
```

Comment:
Foreslår å rename denne til bare `labels`. Cascading høres kult ut, men min erfaring er at når man føler seg smart er det et rødt flagg for kompleksitet :) 

```suggestion
  labels:
```

Reply:
Labels kan høres ut som noe man bare setter på Application da, a la `application.metadata.labels`

Reply:
Mener det er innafor når dette står i `spec` og `labels` settes vanligvis i `metadata`. De må uansett sjekke dokumentasjonen for å skjønne hva `cascading` betyr, så der kan vi skrive som det står i dag at dette feltet setter labels på alle application-genererte ressurser

Reply:
Poeng det :)